hadoop-hdds/docs/content/integration/DistCp.md

```diff
@@ -0,0 +1,76 @@
+---
+title: DistCp
+weight: 4
+menu:
+   main:
+      parent: "Application integrations"
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# Hadoop DistCp
+
+The Hadoop DistCP is a command line, MapReduce-based tool for bulk data copying.
```

Comment:
We can link to the hadoop docs right at the top instead of waiting until the end of the document. We may want to still keep the last sentence in the doc as having the link too for clarity.
```suggestion
[Hadoop DistCP](https://hadoop.apache.org/docs/current/hadoop-distcp/DistCp.html) is a command line, MapReduce-based tool for bulk data copying.
```